daemon/links/links_test.go

```diff
@@ -197,16 +197,16 @@ func TestLinkPortRangeEnv(t *testing.T) {
 		tcpport := fmt.Sprintf("DOCKER_PORT_%d_TCP+PORT", i)
 		tcpproto := fmt.Sprintf("DOCKER_PORT_%d_TCP+PROTO", i)
 		tcp := fmt.Sprintf("DOCKER_PORT_%d_TCP", i)
-		if env[tcpaddr] == "172.0.17.2" {
+		if env[tcpaddr] != "172.0.17.2" {
```

Comment:
Looked like this was there for a long time; https://github.com/docker/docker/commit/611a23aa7f443284fbbbb80716c46159e438bf52